ui/src/app/dashboard/views/DiscoveriesList/DiscoveriesList.tsx

```diff
@@ -1,16 +1,75 @@
+import { useEffect } from "react";
+
+import { useSelector, useDispatch } from "react-redux";
+
 import DiscoveriesListHeader from "../DiscoveriesListHeader";
 
 import Section from "app/base/components/Section";
 import { useWindowTitle } from "app/base/hooks";
+import { actions } from "app/store/discovery";
+import discoverySelectors from "app/store/discovery/selectors";
 
 const DiscoveriesList = (): JSX.Element => {
   useWindowTitle("Dashboard");
 
+  const discoveries = useSelector(discoverySelectors.all);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(actions.fetch());
+  }, [dispatch]);
+
   return (
     <Section
       header={<DiscoveriesListHeader />}
       headerClassName="u-no-padding--bottom"
-    />
+    >
+      <table>
```

Comment:
You should probably stick with using the `MainTable` component that is used elsewhere in the application.

Reply:
Yeah, give `MainTable` a go. What @bartaz did for the [domains list table](https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/domains/views/DomainsList/DomainsTable/DomainsTable.tsx) is a good place to start.

When the table starts to get a bit more interactive (sorting, actions, etc), you can use the [machine tests table](https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/machines/views/MachineDetails/MachineTests/MachineTestsTable/MachineTestsTable.tsx) or the [DHCP table](https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/machines/views/MachineDetails/MachineNetwork/DHCPTable/DHCPTable.tsx) as guides. There's also things like the [available storage table](https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/machines/views/MachineDetails/MachineStorage/AvailableStorageTable/AvailableStorageTable.tsx), but I don't think the discoveries list will get to that level of complexity.